cmd/relay/main.go

```diff
@@ -0,0 +1,494 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/bluesky-social/indigo/atproto/identity"
+	"gorm.io/gorm"
+	"io"
+	"log/slog"
+	_ "net/http/pprof"
+	"net/url"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	libbgs "github.com/bluesky-social/indigo/cmd/relay/bgs"
+	"github.com/bluesky-social/indigo/cmd/relay/events"
+	"github.com/bluesky-social/indigo/cmd/relay/repomgr"
+	"github.com/bluesky-social/indigo/util"
+	"github.com/bluesky-social/indigo/util/cliutil"
+	"github.com/bluesky-social/indigo/xrpc"
+
+	_ "github.com/joho/godotenv/autoload"
+	_ "go.uber.org/automaxprocs"
+
+	"github.com/carlmjohnson/versioninfo"
+	"github.com/urfave/cli/v2"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	"gorm.io/plugin/opentelemetry/tracing"
+)
+
+func init() {
+	// control log level using, eg, GOLOG_LOG_LEVEL=debug
+	//logging.SetAllLoggers(logging.LevelDebug)
+}
+
+func main() {
+	if err := run(os.Args); err != nil {
+		slog.Error(err.Error())
+		os.Exit(1)
+	}
+}
+
+func run(args []string) error {
+
+	app := cli.App{
+		Name:    "relay",
+		Usage:   "atproto Relay daemon",
+		Version: versioninfo.Short(),
+	}
+
+	app.Flags = []cli.Flag{
+		&cli.BoolFlag{
+			Name: "jaeger",
+		},
+		&cli.StringFlag{
+			Name:    "db-url",
+			Usage:   "database connection string for BGS database",
+			Value:   "sqlite://./data/bigsky/bgs.sqlite",
```

Comment:
is there still sqlite support? I thought that got dropped

Reply:
it's not working right now because some transactions changed, but it's maybe not far off from being fixed?

Reply:
I would at least update the default from `.../bigsky/...` to `relay` so dev databases don't get clobbered in a weird way